### PR TITLE
Disable exception tracking via profiler

### PR DESCRIPTION
**Pull request by jander-msft**
Remove exception tracking while evaluating future designs and allow the profiler to ship without it.

### CODE REVIEW

**Review by wiktork — DISMISSED** [review]
Did you want to disable this with a preprocessor directive, to make it easier to flip?

File: src/MonitorProfiler/CMakeLists.txt [hunk 1]
```diff
@@ -38,6 +38,9 @@ set(SOURCES
     Communication/CommandServer.cpp
     )
 
+# Include exceptions tracking feature
+#add_compile_options(-DDOTNETMONITOR_FEATURE_EXCEPTIONS)
```
Comment (kkeirstead) [inline src/MonitorProfiler/CMakeLists.txt:42]:
```suggestion
#add_compile_options(-DOTNETMONITOR_FEATURE_EXCEPTIONS)
```

Comment (jander-msft) [reply src/MonitorProfiler/CMakeLists.txt:42]: It is correct as-is. The `-D` specifies that it is a preprocessor definition.

**Review by wiktork — APPROVED** [review]

**Pull request merged**